src/refactor_nrepl/core.clj

```diff
@@ -152,28 +152,28 @@
 
 (defn read-ns-form
   ([path]
-   (let [^String path-string (when (string? path)
```

Comment:
DRYed out the duplication across the two arities of `read-ns-form`.

Took care of making them 100% equivalent. The key is here:

```clj
(if dialect
  {:read-cond :allow :features #{dialect}}
  nil)
```

The nil value makes `readers/indexing-push-back-reader` behave exactly as if it hadn't received this argument at all.